src/rules/cyclomaticComplexityRule.ts

```diff
@@ -90,7 +90,7 @@ function walk(ctx: Lint.WalkContext<{ threshold: number }>): void {
     let complexity = 0;
 
     return ts.forEachChild(ctx.sourceFile, function cb(node: ts.Node): void {
-        if (isFunctionScopeBoundary(node)) {
+        if (isFunctionScopeBoundary(node) !== ScopeBoundary.None) {
```

Comment:
This is due to a breaking change in tsutils API.
https://github.com/ajafff/tsutils/releases/tag/v3.0.0